spec/support/contexts/payout_context.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+# License: AGPL-3.0-or-later WITH WTO-AP-3.0-or-later
+# Full license explanation at https://github.com/houdiniproject/houdini/blob/main/LICENSE
+
+shared_examples 'payout that can create object events with #publish_created' do
+  let(:instance) { subject }
+
+  it {
+    initial_count = ObjectEvent.all.count
+    instance.publish_created
+    expect(ObjectEvent.all.count).to eq(initial_count + 1)
+    }
+end
```

Comment:
I think this specific example should be handled placed into `payout_spec.rb`. Generally, shared examples are most useful in certain specific cases where you have to check the same characteristics on different  types of objects but none of the input is derived from code you're running. For example, if you look at

```ruby
it_behaves_like 'an houidable entity', :np
```

This makes sense because you're checking the same thing on a bunch of different classes and it has minimal input.

Do you foresee reusing this example elsewhere?

Reply:
Honestly, probably not. Willing to move it to `payout_spec`